pom.xml

```diff
@@ -99,7 +99,12 @@
     <dependency>
       <groupId>com.amazonaws</groupId>
       <artifactId>amazon-kinesis-client</artifactId>
-      <version>1.8.10</version>
+      <version>1.9.0</version>
+    </dependency>
+    <dependency>
+    <groupId>com.amazonaws</groupId>
+      <artifactId>aws-java-sdk-core</artifactId>
+      <version>1.11.430</version>
```

Comment:
We are using version 1.11.271 for `aws-java-sdk-sts`  shall we use the same version at both places?

Reply:
+1

Reply:
271 seems to suffer from [this bug](https://github.com/aws/aws-sdk-java/issues/1611).

Reply:
It worked with 307 instead